### ISSUE TITLE
Syntax highlighting broken for object parameters

### ISSUE THREAD

**Issue by alem0lars** [HerringtonDarkholme/yats.vim#34]
![2018-01-17-083733_1080x467_scrot](https://user-images.githubusercontent.com/336957/35031068-b986b9e2-fb61-11e7-9684-495a8944bf40.png)

**Comment by HerringtonDarkholme** [comment]
I don't see anything wrong here.
Please specify which part is broken.

**Comment by alem0lars** [comment]
The syntax highlighting of:

```
{
  outputStream = process.stdout,
  errorStream = process.stderr,
}: { outputStream: WritableStream; errorStream: WritableStream }
```

**Comment by HerringtonDarkholme** [comment]
They are correctly highlighted. 

The first `outputStream` is parameter name binding, and it is highlighted the same as `env`. This is intended.

The second `outputStream` is highlighted as field in type which is also correct.

**Comment by alem0lars** [comment]
If you write `process.stdout` in normal function code it gets highlighted differently. Am I wrong?

**Comment by HerringtonDarkholme** [comment]
Fixed in 0bf025541b8a192c6c954911ab6460a052ad22ab
